pages/windows/slmgr.md

```diff
@@ -0,0 +1,37 @@
+# slmgr
+
+> Install, activate, and manage Windows licenses.
+> This command may override, deactivate, and/or remove your current Windows license. Please proceed with caution.
+> More information: <https://docs.microsoft.com/windows-server/get-started/activation-slmgr-vbs-options>.
+
+- [D]isplay the current Windows [l]icense [i]nformation:
```

Comment:
```suggestion
- Display the current Windows [l]icense [i]nformation:
```
If options are case insensitive everything is right, otherwise please address this comment. The same applies for other similar comments.

Reply:
We could use `[d]isplay` tough. We do that in quite a few pages.

Reply:
I am not sure what we have to choose here. If there is no formal rule for this it must be written. :)

Reply:
Windows command-line programs commonly adopt one of these three different parameter conventions:

+ The convention deriving from DOS where all parameters are used using forward slashes, such as `cd {{path/to/directory}} /d` to switch directories. `slmgr.vbs` uses this convention.
+ The Unix-like convention as accepted by many major, cross-platform apps, such as `firefox --profile profile1` and `git --help`.
+ The PowerShell convention where long parameters are also using single dash instead of 2 in Unix, like `Invoke-WebRequest -Method Post -Body @{ name='bob' } http://example.com/form`

The usage of mnemonic hints here is intended as some commands, including Windows commands which are derived from the DOS days, often do not include long parameter formats (such as `--help` instead of `-h`) which [we encourage writers to do so](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).

I also encourage the use of that for #8200 as well. Maybe we should open a separate issue about standardizing these hints.

Reply:
> while [we encourage writers to use long parameter formats as possible](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).

Unfortunately there is no formal rules written for this now. It's better to write rules for everything whenever possible because new comers can be confused when we require them to use long options when there is no such requirement in style guide. 😄 I think I can fill this space in guides.

Reply:
Already created an issue for that in #8304